tests/playwright/config/sanity.config.js

```diff
@@ -6,15 +6,24 @@ const config = {
 	timeout: 40000,
 	globalTimeout: 900000,
 	reporter: 'list',
-	testDir: '../sanity',
+	testDir: '../sanity/',
+	testMatch: [
+		'modules/*test.*js',
+		'*test.*js',
+	],
 	globalSetup: require.resolve( './global-setup' ),
 	retries: 1,
 	use: {
 		headless: true,
-		storageState: './tests/playwright/config/storageState.json',
+		storageStatePath: '/tmp/elementor/playwright/storageState.json',
+		storageState: '/tmp/elementor/playwright/storageState.json', // Save the login state, reduce the requirements to login after 'globalSetup' runs.
 		baseURL: 'http://localhost:8888/',
 		viewport: { width: 1440, height: 960 },
 		video: 'on-first-retry',
+		user: {
+			username: 'admin',
+			password: 'password',
```

Comment:
Check the env e.g. `env.username || 'admin'`

Reply:
Handled.